content/java/concepts/calendar/terms/getDisplayName/getDisplayName.md

```diff
@@ -0,0 +1,87 @@
+---
+Title: '.getDisplayName()' 
+Description: 'Returns the string representation (display name) of the calender field value in the given style and locale. If no string representation is applicable, null is returned.' 
+Subjects: 
+  - 'Code Foundations'
+  - 'Computer Science'
+ 
+Tags: 
+  - 'Date'
+  - 'Methods'
+  - 'Calender'
+CatalogContent: 
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+
+The **.getDisplayName()** method of the `Calender` class returns the string representation (display name) of the calender field value in the given style and locale. If no string representation is applicable, null is returned.
+
+## Syntax
+
+```pseudo
+public String getDisplayName(int field, int style, Locale locale)
+```
+
+```pseudo
+// Create a Calendar object.
+Calendar calendar = Calendar.getInstance();
+
+// Get the display name for the month in the current locale.
+String monthName = calendar.getDisplayName(Calendar.MONTH, Calendar.SHORT, Locale.getDefault());
+
+// Print the month name.
+System.out.println(monthName);
+```
+
+## Parameters
+
+- `field`: The first parameter field, specifies the calender field for which the display name is returned. Possible values for the field parameter include: `Calender.MONTH`, `Calender.DAY_OF_WEEK`, `Calender.YEAR`.
+
+- `style`: The second parameter, style, specifies the style of the display name. Possible values for the style parameter include: `Calender.SHORT` and `Calender.LONG`. Both values can either return a short or long form of the display name.
+
+- `locale`: The third parameter, locale, specifies the locale for which the display name is returned. Possible values for the field parameter include: `Locale.ENGLISH`, `Locale.FRENCH`, `Locale.GERMAN`.
```

Comment:
Here, you should use the variables, and parameters from syntax to further explain them. For example:
- `calendarInstance`: Instance of a calendar

Reply:
Sorry about that, working on it now.